src/annotator/util/clean-context.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Export clean versions of standard functions and objects.
+ *
```

Comment:
I think we need to be a bit clearer about the motivation here and what "clean" means. Specifically that this module is a workaround for environments that overwrite properties on global objects with broken polyfills or other problematic values.